CONTRIBUTING.md

```diff
@@ -104,41 +103,48 @@ variables:
 
```

Comment:
Done in the `Slow Tests` section. Not sure how this comment ended up here \*shrug\*